CONTRIBUTING.md

```diff
@@ -29,41 +29,58 @@ Pull requests are welcome.
 - You need Python 3.8 installed.
   - This is the minimum supported version of the tool.  Developing with a later version risks introducing type hints such as `list[dict]` that are not compatible with Python 3.8.
 
-## Running without installing
+Initialise Poetry by telling it where Python 3.8 is, e.g.
 
-Use `run.py`.  E.g.
 ```
-python run.py ls "C:\temp\sarif_files"
+# Windows - adjust to the path where you have installed Python 3.8.
+poetry env use "C:\Python38\python.exe"
+# Linux
+poetry env use 3.8
```

Comment:
Is it worth stating here (for linux at least), that this is only required if your default python isn't 3.8?

Reply:
I've added that comment